app/controllers/auth0_controller.rb

```diff
@@ -0,0 +1,83 @@
+class Auth0Controller < ApplicationController
```

Comment:
The `logout_url` method in `Auth0Controller` is using string interpolation to build the query string, which can lead to URL manipulation vulnerabilities. Consider using `URI.encode_www_form` or similar methods to safely encode query parameters.